### PR TITLE
Deploy to ephemeral env in pr_check

### PR DESCRIPTION
**Pull request by bmclaughlin**
~~Draft PR until #776, #786 are merged.~~ Merged.

Issue: AAH-582

### CODE REVIEW

**Comment by alikins** [comment]
Looks ok to me, assuming `$CICD_ROOT/deploy_ephemeral_env.sh` exists.

But I know nothing of clowder, so not a review.

File: openshift/clowder/clowd-app.yaml [hunk 1]
```diff
@@ -1,82 +1,88 @@
-apiVersion: cloud.redhat.com/v1alpha1
-kind: ClowdApp
+apiVersion: v1
+kind: Template
 metadata:
   name: automation-hub
-  namespace: automation-hub
-spec:
-  # The name of the ClowdEnvironment providing the services
-  envName: automation-hub-env
```
Comment (jameswnl) [inline openshift/clowder/clowd-app.yaml:8]: you may want to use `${ENV_NAME}` for the value and add a parameter `ENV_NAME` which will be assigned by e.g. the prod env value here https://gitlab.cee.redhat.com/service/app-interface/blob/master/data/products/insights/environments/production.yml#L53

File: openshift/clowder/clowd-app.yaml [hunk 2]
```diff
@@ -1,82 +1,88 @@
...
-  # Creates a Service on port 8000
-  webServices:
-    public: true
-    private: false
-    metrics: false
+    # The bulk of your App. This is where your running apps will live
+    deployments:
+    - name: galaxy-api
+      podSpec:
+        image: quay.io/cloudservices/automation-hub-galaxy-ng
```
Comment (jameswnl) [inline openshift/clowder/clowd-app.yaml:20]: You would want to parameterized the image tag e.g. 
`image: quay.io/cloudservices/automation-hub-galaxy-ng:${IMAGE_TAG}`

File: openshift/clowder/clowd-app.yaml [hunk 3]
```diff
@@ -1,82 +1,88 @@
...
+          - name: PULP_CONTENT_ORIGIN
+            value: 'localhost'
 
-  # Creates a database if local mode, or uses RDS in production
-  database:
-    # Must specify both a name and a major postgres version
-    name: automation-hub
-    version: 12
+    # Creates a Service on port 8000
+    webServices:
```
Comment (jameswnl) [inline openshift/clowder/clowd-app.yaml:74]: `webServices` is per `deployment` as in https://redhatinsights.github.io/clowder/clowder/dev/api_reference.html#k8s-api-cloud-redhat-com-clowder-v2-apis-cloud-redhat-com-v1alpha1-webservices

So in one clowd app, you can have multiple webservices

**Review by jameswnl — APPROVED** [review]
This is good to go.

Will need to implement health check probes in the api code in a fellowup PR.

**Review by cutwater — APPROVED** [review]

**Pull request merged**